phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -418,7 +420,7 @@ public abstract class BaseTest {
     }
     
     private static final String ORG_ID = "00D300000000XHP";
-    protected static int NUM_SLAVES_BASE = 1;
+    protected static int NUM_SLAVES_BASE = 3;
```

Comment:
This doesn't look like a good idea.
It would be better to override this on the tests where it is needed

Reply:
@stoty  This PR is not ready for review yet. I was just trying to run all the tests to see which ones fails in the first pass. 
I should have labelled the PR as Draft PR. Sorry for the confusion.